code/game/objects/items/storage/pill_packets.dm

```diff
@@ -66,7 +65,7 @@
 	pip_color = COLOR_PACKET_DYLOVENE
 
 /obj/item/storage/pill_bottle/packet/paracetamol
-	name = "paracetamol pill packet"
+	name = "paracematol pill packet"
```

Comment:
Probably not a good idea to re-introduce typos

Reply:
yous a tipo

Reply:
a result of non-atomic changes